### ISSUE TITLE
Cursor not showing on a WebRTC screenshare

### ISSUE THREAD

**Issue by techvio1** [filips123/PWAsForFirefox#171]
### Description

When starting a stream, the cursor does not show on PWA Apps, while on normal firefox, the cursor shows with no issue, this happens on any WebRTC screenshare.

### Steps to Reproduce

1. Go to Discord PWA App (Or any WebRTC PWA App)
2. Start a stream
3. Choose display on pipewire
4. Stream starts
5. Trying to move the cursor
6. Cursor does not move (on stream)

### Additional Information

There is nothing I found about it except that the stream works fine on normal Firefox, and when pointing to a normal firefox, the cursor shows, but not on other apps

### Affected Websites

*    discord.com/app
*    basically any website support WebRTC

### Environment

* Operating system: Arch Linux
* System architecture: x86_64
* Desktop environment:  KDE Plasma (Wayland)
* Installation method:  AUR
* PWAsForFirefox extension version: 1.4.2
* PWAsForFirefox native version: 1.4.2
* Firefox version: 101.0.1

**Comment by filips123** [comment]
Does it also happen in Firefox you can download [directly from Mozilla website](https://www.mozilla.org/firefox/all/#product-desktop-release)?

**Comment by techvio1** [comment]
After testing with the official Firefox binary, it's now working flawlessly

**Comment by techvio1** [comment]
After testing again, I found out that the issue shows when enabling "Use Wayland Display Server".
